blocks/testimonials/testimonials.js

```diff
@@ -0,0 +1,51 @@
+import {
+  div,
+  blockquote,
+  p,
+  cite,
+} from '../../scripts/dom-helpers.js';
+
+function parseTestimonial(text) {
+  const lastDashIndex = text.lastIndexOf(' - ');
+  if (lastDashIndex === -1) {
+    return { quoteText: text.trim(), name: '', year: '' };
+  }
+
+  const quoteText = text.substring(0, lastDashIndex).trim();
+  const authorPart = text.substring(lastDashIndex + 3).trim();
+
+  const [name, year] = authorPart.split('|').map((s) => s.trim());
+
+  return { quoteText, name, year };
+}
+
+export default function decorate(block) {
+  const quotes = block.querySelectorAll('div > div');
+  const quotesList = div({ class: 'testimonials-grid' });
+  const uniqueQuotes = new Set();
```

Comment:
I'm a bit surprised that we are creating a set here at all.  Why not just bang out the rows as given.

Reply:
I had originally implemented multiple columns and had a bug with repeated quotes. After reverting back to a single column, I never changed the implementation. Though, I don't think this will cause any performance issues

Reply:
fixed the underlying bug